### PR TITLE
Add marketing integrations section

### PR DESCRIPTION
**Pull request by erjohnson**
- Remove unused SSO integration tutorial template
- Add integrations index page
- Add marketing integrations index page
- Add adobe campaign integration tutorial
- Add mailchimp integration tutorial
- Add marketo integration tutorial
- Add images for tutorials
- Update sidebar navigation
- Add new pages to update feed

### CODE REVIEW

File: articles/integrations/marketing/adobe-campaign/index.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+## Import Users to Adobe Campaign
+
+To import your Auth0 users into Adobe Campaign:
+
+- Export your user data as a CSV file with the [User Import / Export Extension](/extensions/user-import-export).
+- Import the file into Adobe Campaign with the [Adobe Campaign Import Wizard](https://docs.campaign.adobe.com/doc/AC/en/PTF_Importing-Exporting_data_Importing_data.html).
+
+::: note
+[Adobe Campaign Documentation: Importing Data](https://docs.campaign.adobe.com/doc/AC/en/PTF_Importing-Exporting_data_Importing_data.html)
```
Comment (mpaktiti) [inline articles/integrations/marketing/adobe-campaign/index.md]: You have the same link on the previous bullet, why add it also as a note?

File: articles/integrations/marketing/adobe-campaign/index.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+`family_name` | Last Name
+
+![User Import/Export Extension Fields](/media/articles/integrations/marketing/import-export-fields.png)
+
+After adding the user fields, click on the **Export Users** button to start the export. Once the export is complete, download the CSV file to use in the following section.
+
+### Import a User Data File
+
+::: note
+[Adobe Campaign Documentation: Importing Data](https://docs.campaign.adobe.com/doc/AC/en/PTF_Importing-Exporting_data_Importing_data.html)
```
Comment (mpaktiti) [inline articles/integrations/marketing/adobe-campaign/index.md:40]: Same here, you have already listed this resource earlier

File: articles/integrations/marketing/mailchimp/index.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+## Import Users to MailChimp
+
+To import your Auth0 users into MailChimp:
+
+- Export your user data as a CSV file with the [User Import / Export Extension](/extensions/user-import-export).
+- Import the file into MailChimp on the [MailChimp Dashboard](https://login.mailchimp.com/).
+
+::: note
+[MailChimp Knowledge Base: Import Subscribers to a List](https://kb.mailchimp.com/lists/growth/import-subscribers-to-a-list)
```
Comment (mpaktiti) [inline articles/integrations/marketing/mailchimp/index.md]: Resource is listed twice in this doc, leave only one

File: articles/integrations/marketing/marketo/index.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+## Import Users to Marketo
+
+To import your Auth0 users into Marketo:
+
+- Export your user data as a CSV file with the [User Import / Export Extension](/extensions/user-import-export).
+- Import the file into Marketo using the [Bulk Leads endpoint](http://developers.marketo.com/rest-api/endpoint-reference/lead-database-endpoint-reference/#/Bulk_Leads) of the Marketo REST API.
+
+::: note
+[Marketo Documentation: Bulk Lead Import](http://developers.marketo.com/rest-api/bulk-import/bulk-lead-import/)
```
Comment (mpaktiti) [inline articles/integrations/marketing/marketo/index.md]: Resource listed twice, leave only one

File: articles/integrations/marketing/adobe-campaign/index.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
+---
```
Comment (mpaktiti) [inline articles/integrations/marketing/adobe-campaign/index.md:1]: Is there a reason why you created separate folders for this? You could name your doc `articles/integrations/marketing/adobe-campaign.md` and the end result would be the same.

It's not a problem but you could have a simpler structure here (unless you plan to add more docs in the `adobe-campaign` folder). Same applies to all 3 marketing docs.

File: updates/2017-11-22.yml [hunk 6]
```diff
@@ -0,0 +1,27 @@
+added:
```
Comment (mpaktiti) [inline updates/2017-11-22.yml:1]: I would also add here the new integrations landing page

**Review by mpaktiti — CHANGES_REQUESTED** [review]

**Comment by mpaktiti** [comment]
@erjohnson Can you also rebase with master? The heroku review app does not work and maybe that will solve it.

File: articles/integrations/marketing/adobe-campaign/index.md (hunk 5 above)
Comment (erjohnson) [reply articles/integrations/marketing/adobe-campaign/index.md:1]: There's features slated for the near future which will probably require additional pages for each integration.

**Pull request merged**